src/elements/block/core-heading/core-heading.spec.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { CoreHeading } from '.'
+
+it('should render with the correct text', () => {
+  const { getByText } = render(
+    <CoreHeading
+      {...{
+        name: 'core/heading',
+        attributes: {
+          align: 'left',
+          content: 'The parables from Matthew',
+          level: 2
+        }
+      }}
```

Comment:
Try not use the spread here when JSX will do.

```suggestion
      name="core/heading"
	  attributes={{
        align: 'left',
        content: 'The parables from Matthew',
        level: 2
	  }}
```